unlock-app/src/components/content/EventContent.tsx

```diff
@@ -0,0 +1,201 @@
+import React from 'react'
+import Head from 'next/head'
+import { Button, Tooltip } from '@unlock-protocol/ui'
+import { pageTitle } from '../../constants'
+import { useRouter } from 'next/router'
+import { AppLayout } from '../interface/layouts/AppLayout'
+import { useMetadata } from '~/hooks/metadata'
+import LoadingIcon from '../interface/Loading'
+import { AddressLink } from '../interface/AddressLink'
+import { ReactMarkdown } from 'react-markdown/lib/react-markdown'
+import { FaCalendar, FaRegCalendarPlus, FaClock } from 'react-icons/fa'
+import { GoLocation } from 'react-icons/go'
+import { FiTwitter } from 'react-icons/fi'
+import Link from 'next/link'
+import { useAuth } from '~/contexts/AuthenticationContext'
+
+const formatEventData = (metadata: any) => {
```

Comment:
Don't we have a function to format data into a standard form we can use already?

I think we can reuse it or make it more reusable if it isn't.

Reply:
Ho Maybe we do. Good catch! I;ll find it!

Reply:
I could not find this anywhere else!

Reply:
https://github.com/unlock-protocol/unlock/blob/master/unlock-app/src/components/interface/locks/metadata/utils.ts

Reply:
Thx! Fixed!